### PR TITLE
Implement more parameters

### PR DESCRIPTION
**Pull request by NicolasBonduel**
I implemented the other parameters from the Gravatar documentation here: https://fr.gravatar.com/site/implement/images

I also updated the tests and the README.

### CODE REVIEW

**Comment by NicolasBonduel** [comment]
Thanks apie, I forgot about this pull request =)

**Comment by apie** [comment]
Hmm, seems the test is now failing for only some versions of Python..

**Comment by NicolasBonduel** [comment]
I'm going to look at it later this week. Also I will take a look at the code quality decrease on landscape.

**Comment by NicolasBonduel** [comment]
Did you remove the landscape check @gfabricio ?

**Comment by gfabrizio** [comment]
Sorry for the delay, actually the landscape check is still there ( https://github.com/gfabricio/gravtr/blob/master/README.md)

btw, lgtm for me. Can I merge and upload a new version or do you have any other considerations?

File: gravtr/__init__.py [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+        X = 'x'
+
     def __init__(self, email):
-        self.email = email
+        self.email = email.encode('utf-8')
 
-    def generate(self, size=None):
-        email = self.email.encode('utf-8')
-        self.url = GRAVATAR_URL + hashlib.md5(email).hexdigest()
+    def generate(self, unsecure=False, size=None, typed=False, default=None, force_default=False, rating_type=None):
```
Comment (NicolasBonduel) [inline gravtr/__init__.py:21]: Landscape / Codacy (well, really PyLint) is going to complain about this function having too many parameters.
Should we build I build it a different way or just add an exception?

**Comment by NicolasBonduel** [comment]
Other than my previous comment, I don't have any objection.

**Pull request merged**